File: sdkwork-ai-sdk-java/src/main/java/com/sdkwork/ai/model/ChatMessageList.java
package com.sdkwork.ai.model;

public class ChatMessageList {
    private String object;
    private List<ChatMessage> data;
    private Boolean hasMore;
    private String firstId;
    private String lastId;

    public String getObject() {
        return this.object;
    }
    
    public void setObject(String object) {
        this.object = object;
    }

    public List<ChatMessage> getData() {
        return this.data;
    }
    
    public void setData(List<ChatMessage> data) {
        this.data = data;
    }

    public Boolean getHasMore() {
        return this.hasMore;
    }
    
    public void setHasMore(Boolean hasMore) {
        this.hasMore = hasMore;
    }

    public String getFirstId() {
        return this.firstId;
    }
    
    public void setFirstId(String firstId) {
        this.firstId = firstId;
    }

    public String getLastId() {
        return this.lastId;
    }
    
    public void setLastId(String lastId) {
        this.lastId = lastId;
    }
}

File: sdkwork-ai-sdk-typescript/src/types/chat-message-list.ts
import type { ChatMessage } from './chat-message';

/** OpenAI标准聊天消息列表 */
export interface ChatMessageList {
  /** 对象类型 */
  object?: string;
  /** 消息列表 */
  data?: ChatMessage[];
  /** 是否有更多数据 */
  has_more?: boolean;
  /** 第一条记录ID */
  first_id?: string;
  /** 最后一条记录ID */
  last_id?: string;
}
